test/karma/tsconfig.json

```diff
@@ -35,7 +36,10 @@
     "test-app/**/*.tsx",
     "test-app/**/*.ts",
     "test-app/util.ts",
-    "test-app/global.ts"
+    "test-app/global.ts",
+    "test-invisible-prehydration/components.d.ts",
```

Comment:
I _think_ these would be covered by the `tsconfig` file in the `test-invisible-prehydration` directory and can be removed